### PR TITLE
[APPS-3459] Update en.yml file with marketplace listing content

### PR DESCRIPTION
**Pull request by pmgrove**
:v:

/cc @zendesk/vegemite, @zendesk/i18n 

### Description

Update en.yml file with marketplace listing content

### References
* JIRA: https://zendesk.atlassian.net/browse/APPS-3459

### Risks
* [low] - app content may not show on marketplace

### CODE REVIEW

File: translations/en.yml [hunk 1]
```diff
@@ -6,6 +6,111 @@ packages:
...
+If you need to give an important message to agents, put some instructions about filling out a ticket, or add a link to your retail system, this app enables you to do it.
+
+
+It also has \"data placeholder\" support, which means you can dynamically place data into the text you want to display. This is useful if you want to make a link that includes parameters about the current ticket, requester, or user.
+
+
+By enabling this app, You agree to the [Built by Zendesk Terms of Use](https://www.zendesk.com/company/built-by-zendesk-agmt/)."
+  - translation:
+      key: "txt.apps.text_app.app.installation_instructions"
+      title: "Installation Instructions for App: Text"
```
Comment (injuhwang) [inline translations/en.yml]: Please add the following:
> Do not translate the placeholders (e.g., `ticket.id`).

**Comment by pmgrove** [comment]
@injuhwang 💇

**Comment by injuhwang** [comment]
👍

File: translations/en.yml [hunk 2]
```diff
@@ -6,6 +6,111 @@ packages:
...
+
+
+It also has \"data placeholder\" support, which means you can dynamically place data into the text you want to display. This is useful if you want to make a link that includes parameters about the current ticket, requester, or user.
+
+
+By enabling this app, You agree to the [Built by Zendesk Terms of Use](https://www.zendesk.com/company/built-by-zendesk-agmt/)."
+  - translation:
+      key: "txt.apps.text_app.app.installation_instructions"
+      title: "Installation Instructions for App: Text. Do not translate the placeholders (e.g., ticket.id)."
+      value: "\"Text is a very simple app, so it's really simple to get going.
```
Comment (injuhwang) [inline translations/en.yml]: There's no closing quotation mark.

File: translations/en.yml [hunk 3]
```diff
@@ -6,6 +6,111 @@ packages:
...
+#### Setting up in Zendesk Support
+
+
+1. Sign in to your Zendesk Support account, then click the Admin icon in the sidebar.
+
+2. Select Apps > Marketplace, then find the Text app and click the tile
+
+3. On the Text app page, click Install app in the upper-right corner.
+
+4. Enter a title to display in the \"\"Apps\"\" tray on a ticket.
```
Comment (injuhwang) [inline translations/en.yml]: `Apps` has two double quotation marks.

File: translations/en.yml [hunk 4]
```diff
@@ -6,6 +6,111 @@ packages:
...
+
+
+These are small snippets of text which expand or transform themselves into full strings. For example, to display the current user's ID in your text, add `{{current_user.id}}` in the Text app, and it will display as the current user's ID (`2313`, for example).
+
+
+Also, you could put this into a URL as a parameter:
+
+
+
+   // This is the text I would put in the settings:
```
Comment (injuhwang) [inline translations/en.yml]: Should this also be inside code block?

File: translations/en.yml [hunk 5]
```diff
@@ -6,6 +6,108 @@ packages:
...
+
+
+It also has \"data placeholder\" support, which means you can dynamically place data into the text you want to display. This is useful if you want to make a link that includes parameters about the current ticket, requester, or user.
+
+
+By enabling this app, You agree to the [Built by Zendesk Terms of Use](https://www.zendesk.com/company/built-by-zendesk-agmt/)."
+  - translation:
+      key: "txt.apps.text_app.app.installation_instructions"
+      title: "Installation Instructions for App: Text. Do not translate the placeholders (e.g., ticket.id)."
+      value: "Text is a very simple app, so it's really simple to get going.
```
Comment (injuhwang) [inline translations/en.yml]: Is this `"` at the beginning needed?

**Comment by injuhwang** [comment]
👍 for the strings

File: translations/en.json [hunk 6]
```diff
@@ -1,5 +1,21 @@
...
+    "short_description": {
+      "title": "Description for App: Text",
+      "value": "Any text you want, with data placeholders"
+    },
+    "long_description": {
+      "title": "Long Description for App: Text",
+      "value": "Text enables you to put any text you want on a ticket page.\n\nIf you need to give an important message to agents, put some instructions about filling out a ticket, or add a link to your retail system, this app enables you to do it.\n\nIt also has \"data placeholder\" support, which means you can dynamically place data into the text you want to display. This is useful if you want to make a link that includes parameters about the current ticket, requester, or user.\n\nBy enabling this app, You agree to the [Built by Zendesk Terms of Use](https://www.zendesk.com/company/built-by-zendesk-agmt/)."
+    },
+    "installation_instructions": {
+      "title": "Installation Instructions for App: Text. Do not translate the placeholders (e.g., ticket.id).",
```
Comment (svizzari) [inline translations/en.json]: nit: Worth mentioning that placeholders are within `{{...}}`?

Comment (pmgrove) [inline translations/en.json]: the placeholders don't always have `{{...}}` around them

**Pull request merged**